### PR TITLE
[shopsys] Small updates to readme.md file

### PR DESCRIPTION
**Pull request by sslt**
| Q             | A
| ------------- | ---
|Description, reason for the PR| Small updates to readme.md file reflecting changes introduced in current version of SSFW
|New feature| No 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,31 +26,19 @@ Shopsys Framework is fully functional e-commerce platform with all basic functio
...
+* Full friendly URL for main entities and full control over SEO aspects of online presence
 * Performance optimization through Elasticsearch, Redis, PostgreSQL
 * Full core upgradability
 * GDPR compliance
 * Preparation for scalability
-* Base for [Back-end API](https://docs.shopsys.com/en/latest/backend-api/)
+* Basic [Back-end API](https://docs.shopsys.com/en/latest/backend-api/)
 * Manifest for orchestration via [Kubernetes](https://docs.shopsys.com/en/latest/kubernetes/introduction-to-kubernetes/)
 * Support to easier [deployment to Google Cloud via Terraform](https://docs.shopsys.com/en/latest/kubernetes/how-to-deploy-ssfw-to-google-cloud-platform/)
+* Basic GraphQL Front-end API for implementation of own front-end and PWA
```
Comment (grossmannmartin) [inline README.md:37]:
```suggestion
* Basic GraphQL Front-end API for implementation of own front-end and PWA
```

travis failing on this one

File: README.md [hunk 2]
```diff
@@ -26,31 +26,19 @@ Shopsys Framework is fully functional e-commerce platform with all basic functio
...
 * Basic CMS
 * Support for several currencies, [languages, and domains](https://docs.shopsys.com/en/latest/introduction/domain-multidomain-multilanguage/)
-* Full friendly URL for main entities
+* Full friendly URL for main entities and full control over SEO aspects of online presence
 * Performance optimization through Elasticsearch, Redis, PostgreSQL
 * Full core upgradability
 * GDPR compliance
 * Preparation for scalability
-* Base for [Back-end API](https://docs.shopsys.com/en/latest/backend-api/)
+* Basic [Back-end API](https://docs.shopsys.com/en/latest/backend-api/)
```
Comment (TomasLudvik) [inline README.md:34]: I think that Base is more appropriate as there are not many supported functions

Comment (sslt) [reply README.md:34]: In my humble opinion, in this context isn't phrase "Base for" the most suitable, that's why I proposed to change it. Let's make it democratic and wait for third opinion?

Comment (sslt) [reply README.md:34]: After short slack conversation, we voted for "Basic..."

**Pull request merged**